backend/electrum_backend.go

```diff
@@ -345,15 +338,15 @@ func (eb *ElectrumBackend) processTxRequest(node *electrum.Node, txHash string)
 	return nil
 }
 
-func (eb *ElectrumBackend) getTxHeight(txHash string) (int64, error) {
+func (eb *ElectrumBackend) getTxHeight(txHash string) int64 {
 	eb.transactionsMu.Lock()
 	defer eb.transactionsMu.Unlock()
 
 	height, exists := eb.transactions[txHash]
 	if !exists {
-		return -1, fmt.Errorf("no block height information for transaction %s yet", txHash)
+		log.Panicf("transactions cache miss for %s", txHash)
```

Comment:
I'd rather we handled those errors properly rather than panic. We could handle retries properly or even return optional errors all the way to the accounter in the `AddrResponse` or `TxResponse`. The accounter could decide when to retry or just flat out exit - all in one place.

Reply:
this should never happen. I think it's fine to panic.